forcats_tutorial_translation.Rmd

```diff
@@ -0,0 +1,96 @@
+# Forcats Tutorial Translation
+
+Xu Xu and Xiaoyun Zhu
+
+
+```{r setup, include=FALSE}
```

Comment:
this should read {r, include=FALSE}. There should be no "setup" tag in your submission

Reply:
We have fixed the issues above and committed it again.